tests/src/reloadFunctionalTest/kotlin/org/jetbrains/compose/reload/tests/VerificationTests.kt

```diff
@@ -84,11 +84,60 @@ class VerificationTests {
         }
     }
 
+    @HotReloadTest
+    @QuickTest
+    fun `illegal code change - change local variables in main`(fixture: HotReloadTestFixture) = fixture.runTest {
+        val code = fixture.initialSourceCode(
+            """
+                import androidx.compose.material.Text
+                import org.jetbrains.compose.reload.test.*
+                
+                fun main() {
+                    val str1 = "Hello"
+                    screenshotTestApplication {
+                        Text(str1 + "world!")
+                    }
+                }
+            """.trimIndent()
+        )
+
+        fixture.checkScreenshot("0-initial")
+
+        fixture.runTransaction {
+            /*
+            Legal Change: replace the code inside the composable function
+            */
+            code.replaceText(""""world!"""", """" world!"""")
+            requestReload()
+            val request = skipToMessage<OrchestrationMessage.ReloadClassesRequest>()
+            val result = skipToMessage<OrchestrationMessage.ReloadClassesResult>()
```

Comment:
In this case I think there is a 'awaitReload' maybe even a 'reload()' which is 'requestReload' and 'awaitReload'?